packages/blocks/src/store/private-selectors.js

```diff
@@ -32,6 +32,43 @@ const ROOT_BLOCK_SUPPORTS = [
 	'letterSpacing',
 ];
 
+const TYPOGRAPHY_SUPPORTS_EXPERIMENTAL_TO_STABLE = {
+	__experimentalFontFamily: 'fontFamily',
+	__experimentalTextDecoration: 'textDecoration',
+	__experimentalFontStyle: 'fontStyle',
+	__experimentalFontWeight: 'fontWeight',
+	__experimentalLetterSpacing: 'letterSpacing',
+	__experimentalTextTransform: 'textTransform',
+	__experimentalWritingMode: 'writingMode',
+};
+
+// Stabilize the style property support keys by mapping experimental supports to stable supports.
+const STYLE_PROPERTY = Object.keys( RAW_STYLE_PROPERTY ).reduce(
```

Comment:
Note: I've added this here instead of updating the `STYLE_PROPERTY` directly in `api/constants` as unfortunately the constants are exported publicly. If we think it's safe to change, I can update that file directly instead of this part of the change. Here's the original file:

https://github.com/WordPress/gutenberg/blob/02c6a8fa2f8996f1781d4df0e899c73a6de34c5f/packages/blocks/src/api/constants.js#L17-L272